target/scripts/helpers/database/manage/postfix-accounts.sh

```diff
@@ -98,9 +98,14 @@ function __account_already_exists() {
 
 # Also used by addsaslpassword
 function _password_request_if_missing() {
+  local CONFIRM
```

Comment:
Perhaps `PASSWD_CONFIRM` or similar would be a bit better in context. I'm not fussed either way though.